webapp/channels/src/components/youtube_video/youtube_video.tsx

```diff
@@ -75,6 +77,10 @@ export default class YoutubeVideo extends React.PureComponent<Props, State> {
                     height='360px'
                     frameBorder='0'
                     allowFullScreen={true}
+                    allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share'
+                    referrerPolicy={this.props.youtubeReferrerPolicy ? 'strict-origin-when-cross-origin' : undefined}
+                    title={videoTitle}
+                    sandbox='allow-scripts allow-same-origin allow-popups allow-presentation'
```

Comment:
Apart of the referrerPolicy... The rest of these props are needed? Just for my own knowledge, do you mind explaining what they do and how it differ from the default value?

@esarafianou Just in case you have any concern about any of these other props.

Reply:
[I just uses parameters from Slack.](https://github.com/mattermost/mattermost/pull/26980#issuecomment-2155201354) If there are useless parameter and reply it, I'll remove it.